05/DB.test.js

```diff
@@ -1 +1,119 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+it('return 0 when database empty', () => {
+    const db = new DB();
+    expect(db._rows.length).toBe(0);
```

Comment:
Lepiej użyć dostępnej metody tj. `.getRows()` ponieważ znak `_` oznacza (taka konwencja), że to element wew. i nie powinniśmy się do niego odwoływać. Tylko pamiętaj, że ta metoda jest asynchroniczna.